app/server.py

```diff
@@ -41,7 +41,7 @@ def search(search_engine):
         engine = search_engine
         if engine not in ('google', 'bing', 'duckduckgo', 'yahoo', 'ask',
                           'yandex', 'ubaidu', 'exalead', 'quora', 'tyoutube',
-                          'parsijoo', 'mojeek'):
+                          'parsijoo', 'mojeek', 'sstackoverflow'):
```

Comment:
`sstackoverflow`?

Reply:
I added `s` as key for `stackoverflow`  in `__init__` file of Scrapers. So just thought of adding it here in same manner.
Changing it now though :smile: